client/src/plugins/camunda-plugin/deployment-tool/DeploymentTool.js

```diff
@@ -167,16 +171,17 @@ export default class DeploymentTool extends PureComponent {
       duration: 4000
     });
 
+    // If we retrieved the executionPlatformVersion, include it in event
+    const deployedTo = (version &&
+      { executionPlatformVersion: version, executionPlatform: ET_EXECUTION_PLATFORM_NAME }) || undefined;
+
     // notify interested parties
     triggerAction('emit-event', {
       type: 'deployment.done',
       payload: {
         deployment,
-        deployedTo: {
-          executionPlatformVersion: version,
-          executionPlatform: ET_EXECUTION_PLATFORM_NAME
-        },
-        context: 'deploymentTool'
+        context: 'deploymentTool',
+        ...(deployedTo && { deployedTo: deployedTo })
```

Comment:
Missing test coverage here, too.

I think we can safely set `deployedTo` to `null` to indicate we could have set it, but did not retrieve any.

Reply:
This was already in the codebase for `handleDeploymentError`. It omitted the `deployedTo` when there was no `deployedTo` available.